Dockerfile.template

```diff
@@ -115,6 +174,7 @@ RUN set -eux; \
 			ppc64-linux-user ppc64le-linux-user riscv64-linux-user sparc64-linux-user \
 			s390x-linux-user \
```

Comment:
As noted in https://github.com/tianon/docker-qemu/pull/20#issuecomment-803233328, I should test and see if ditching these saves us much space, but that's a problem for another day (as the meme goes, "I am Le Tired" :sleeping:).

Reply:
Yep, saves ~80MB.  That's pretty sizeable.

Perhaps we need a separate `user-static` variant too...  (specifically for binfmt, I guess... :face_with_head_bandage:)